html/src/index.ts

```diff
@@ -40,8 +40,9 @@ export {HTMLSource} from './HTMLSource';
  * @param {Function} effect a callback function that takes a string of rendered
  * HTML as input and should run a side effect, returning nothing.
  * @param {HTMLDriverOptions} options an object with one optional property:
```

Comment:
@ryota-ka Could this line say "options an object with **two** optional properties:" ?

Reply:
oops, I missed this when I edited this file. thanks!